pyproject.toml

```
@@ -43,7 +43,7 @@ google-cloud-secret-manager = "^2.17.0"
 isodate = "~0.6.1"
 Jinja2 = "~3.1.2"
 jsonref = "~0.2"
-jsonschema = "~4.17.3"  # 4.18 has some significant breaking changes: https://github.com/python-jsonschema/jsonschema/releases/tag/v4.18.0
+jsonschema = ">=4.17.3,<5.0"  # Broadened constraint to maintain compatibility while allowing 4.20.0+ for fastmcp
```

If we're successful, this comment is no longer needed.

I'm removing. Pull after 30 seconds or so.

```suggestion
jsonschema = ">=4.17.3,<5.0"
```